qutip/wigner.py

```diff
@@ -35,8 +35,9 @@
            'spin_wigner', 'wigner_transform']
 
 import numpy as np
+import warnings
 from numpy import (zeros, array, arange, exp, real, conj, pi,
-                   copy, sqrt, meshgrid, size, conjugate,
+                   copy, sqrt, meshgrid, size, polyval, fliplr, conjugate,
                    cos, sin)
 from scipy import polyval, fliplr
```

Comment:
How come `polyval` and `fliplr` are imported from both numpy and scipy?  Should probably just be `numpy`.